### ISSUE TITLE
Comment clean up.

### ISSUE THREAD

**Issue by dlemstra** [piql/afs#4]
When going through the code I noticed that at various places there are comments like this:

```c
// If control data pointer is NULL return NULL.
if (control_data == NULL)
{
    return NULL;
}
```

This is stating the obvious and I am wondering if the team would mind me going through the code to remove them? I don't think they add value and some of them are even incorrect:

```c
// If control data pointer is NULL return DFALSE
if (control_data == NULL)
{
    return NULL;
}
```

**Comment by hakonlarsson** [comment]
You're quite right, it doesn't make sense to me either. Please do go ahead and remove them.

**Comment by petterreinholdtsen** [comment]
This issue should have been closed when #10 was merged.
